docs/use-cases.md

```diff
@@ -0,0 +1,19 @@
+# Key Transparency Use Cases
+
+
+Key Transparency ensures, with mathematical certainty, that account owners can
+see all the public keys that have been associated with their account.  
+
+This zero-trust architecture is useful in a variety of scenarios from end-to-end encryption to enterprise account management.
```

Comment:
Is it zero trust, or something a bit weaker? Some sort of "trust but verify" thing?
Just want to make sure we don't promise too much here.

Reply:
Good point. How's "assured system design"?